package.json

```diff
@@ -28,6 +28,7 @@
     "mocha": "^5.2.0"
   },
   "dependencies": {
+    "axios": "^0.21.1",
```

Comment:
Can you please remove axios under devDependencies? I looked through the dev dependency list and the others shouldn't need to be moved.